starter-code/src/viking.js

```diff
@@ -1,11 +1,94 @@
 // Soldier
-function Soldier() {}
+function Soldier(health,strength) {
+    this.health=health;
+    this.strength=strength;
+}
+Soldier.prototype.attack=function(){
+    return this.strength;
+}
+Soldier.prototype.receiveDamage=function(theDamage){
+    this.health=this.health-theDamage;
+    return
+}
 
 // Viking
-function Viking() {}
+Viking.prototype = Object.create(Soldier.prototype);
 
+function Viking (name, health, strength) { 
+    this.name = name; 
```

Comment:
Deben utilizar la herencia de propiedades del contructor padre. Solucion:
function Viking(name, health, strength) {
    this.name = name;
    Soldier.call(this, health, strength);
}